subprojects/docs/src/docs/userguide/build_environment.adoc

```diff
@@ -67,6 +67,9 @@ When set to `all`, `summary` or `none`, Gradle will use different warning type d
 `org.gradle.workers.max=(max # of worker processes)`::
 When configured, Gradle will use a maximum of the given number of workers. Default is number of CPU processors. See also <<command_line_interface.adoc#sec:command_line_performance, performance command-line options>>.
 
+`org.gradle.disable.cache.cleanup=(true,false)`::
+When set to `true`, cache cleanup will be disabled for this gradle home. This property is only settable in the gradle home build.properties file.
```

Comment:
`This property is only settable in the gradle home build.properties file.` reads a little weird. Perhaps ``This property is only read from the `gradle.properties` file in the Gradle home directory.``?